README.md

```diff
@@ -6,7 +6,12 @@ Generate the awesome lists in JSON file.
 # Installation
 
 ```sh
-npm install awesomelists-index
+$ npm install awesomelists-index
+```
+
+Set github token.
+```sh
+$ echo export TOKENS="Your-Github-Token" >> ~/.bash_profile && source ~/.bash_profile
```

Comment:
The way to save github token in `~./bash_profile` is ok I think as we could not get the token again once we close the `access token` page.